src/org/zaproxy/zap/extension/technology/ExtensionTechnology.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
The new API can be added with the extension ExtensionAPI, seems excessive to use a new extension just for this.